scheduler/src/cook/compute_cluster.clj

```diff
@@ -21,6 +21,8 @@
   ; These methods should accept bulk data and process in batches.
   ;(kill-tasks [this task]
   ;(launch-tasks [this offers task-metadata-seq]
+  (ComputeCluster->compute-cluster-name [this])
+
   (compute-cluster->db-id [this])
```

Comment:
This feels like a weird thing to have in the API. Is there any way to avoid this?